ts/src/main/scala/ch/epfl/ts/engine/MarketFXSimulator.scala

```diff
@@ -15,7 +15,7 @@ class MarketFXSimulator(marketId: Long, rules: ForexMarketRules) extends MarketS
     // TODO
 
     case limitAsk: LimitAskOrder =>
-    // TODO
+    // TODO - add limit order resolvation based on given quotes
```

Comment:
In `FullMarketSimulation` I don't use `MarketFXSimulator`. But I implemented something that could be put here in the next PR in `ForexMarketRules`.